Source/WebCore/dom/Document.cpp

```diff
@@ -2322,6 +2322,8 @@ void Document::updateLayoutIgnorePendingStylesheets(Document::RunPostLayoutTasks
             scheduleFullStyleRebuild();
     }
 
+    updateRelevancyOfContentVisibilityElements();
```

Comment:
Can we leave out this change for #16311 ?

Reply:
Removed.